jiant/tasks/lm.py

```diff
@@ -172,3 +177,121 @@ def __init__(self, path, *args, **kw):
             "val": os.path.join(path, "valid.sentences.txt"),
             "test": os.path.join(path, "test.sentences.txt"),
         }
+
+
+class MaskedLanguageModelingTask(LanguageModelingTask):
+    """
+       Generic Masked Language Modeling Task
+    """
+
+    pass
+
+
+@register_task("mlm", rel_path="WikiText103/")
+class MLMTask(MaskedLanguageModelingTask):
+    """
+    Masked language modeling task on Toronto Books dataset
```

Comment:
Maybe add this to the task name? Or specify that we're using data _in the format of_ Toronto Books.

Reply:
Also, what about Wikipedia text?

Reply:
I think this was a copy-pasta error. Fixing now.